README.md

```diff
@@ -28,6 +28,54 @@ You may change this to 'postgres://username:password@localhost:5432/girlscode_li
 ```
 $ npm run dev
 ```
+Open your browser and go to: `http://localhost:3000/`
+
+# Example GitHub workflow
+
+1. Fork this repository by clicking "Fork" on the top right corner
+2. Clone your fork to your machine,
+  `git clone https://github.com/<YOUR GITHUB USERNAME>/libraryapp-hackathon-js-starthere`
+3. Follow the installation steps in this readme file to run the app
+
+## Make changes and save them to your local branch
+
+Let's say you want to add a background image to the project
+
+1. Create a local branch: `git checkout -b addBackgroundImage'
+2. Make the changes and then save
+3. Type `git status` to review what files are being changed
+4. Type `git add .` to add ALL the changed files
+5. Type `git commit -m "Adds a background image"`
+6. Type `git push` to push the changes to your remote
+
+## Remote repository setup
+
+We suggest using two remote repositories:
+- one called upstream, in our case `https://github.com/GirlsCodeMK/libraryapp-hackathon-js-starthere`
+- one called origin, which is your fork `https://github.com/<YOUR GITHUB USERNAME>/libraryapp-hackathon-js-starthere`
+
+You can check your remote(s) by running the following command in your command line / terminal:
+`git remote -v`
+
+You can add a remote by typing: `git add remote origin https://github.com/<YOUR GITHUB USERNAME>/libraryapp-hackathon-js-starthere`
```

Comment:
Change this to 
`git add remote upstream https://github.com/GirlsCodeMK/libraryapp-hackathon-js-starthere`
(the repo will already have an `origin` remote).